gulpfile.js

```diff
@@ -123,6 +124,7 @@ gulp.task("build:bundle", async () => {
       ...esbuildConfig,
       entryPoints: [task],
       outfile: outFilePath,
+      absWorkingDir: path.join(SOURCE_DIR, "common", commonFolder),
```

Comment:
is `absWorkingDir` an esbuild standard field? I can't find it specifically mentioned in the API
https://esbuild.github.io/api/

but I see other plugins use it

Reply:
![image](https://github.com/user-attachments/assets/8bbf35da-136b-448c-9473-9e2de783a6f4)

yee you can see it in the JS API

Reply:
oof that was well hidden 🙏🏻 

https://esbuild.github.io/api/#working-directory (JS tab)